exopy_hqc_legacy/instruments/drivers/visa_tools.py

```diff
@@ -116,13 +119,13 @@ def reopen_connection(self):
         as previously.
 
         """
-        para = {'timeout': self._driver.timeout,
-                'query_delay': self._driver.query_delay,
-                'write_termination': self._driver.write_termination,
-                'read_termination': self._driver.read_termination,
-                }
-        self._driver.close()
-        self.open_connection(**para)
+        # para = {'timeout': self._driver.timeout,
```

Comment:
This is wrong. We need to tolerate an error occurring here but we should not just bypass the whole logic. In particular we should try to close the driver.